lightning/src/util/events.rs

```diff
@@ -520,11 +523,12 @@ impl Writeable for Event {
 					(0, VecWriteWrapper(outputs), required),
 				});
 			},
-			&Event::PaymentForwarded { fee_earned_msat, claim_from_onchain_tx } => {
+			&Event::PaymentForwarded { channel_id, fee_earned_msat, claim_from_onchain_tx } => {
 				7u8.write(writer)?;
 				write_tlv_fields!(writer, {
-					(0, fee_earned_msat, option),
-					(2, claim_from_onchain_tx, required),
+					(0, channel_id, option),
```

Comment:
We can't change the indexes of existing fields or we'll fail to read old data. For new fields, we want to use odd field IDs (which tells old versions to simply ignore the field if they don't understand it) and `option`s (which mean we set the field to `None` if its not included, ie because the object was serialized by an old version of LDK.

Reply:
Changed `channel_id` to 1